### PR TITLE
Amazon Polly Integration

### PR DESCRIPTION
**Pull request by chinazor-allen**
#### What does this PR do?

Allows users to query AWS polly actions.

### CODE REVIEW

File: src/deleteLexicon.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+export default (ctx) => {
+  const { response, logger } = Syncano(ctx);
+
+  const log = logger('Socket scope');
+  const pollyHelper = new helper(ctx.config);
+
+  const deleteLexicon = pollyHelper.deleteLexicon(ctx.args.lexiconName);
+
+  return deleteLexicon
```
Comment (adamwardecki) [inline src/deleteLexicon.js]: you can do `return pollyHelper.deleteLexicon(ctx.args.lexiconName).then(...`

File: test/deleteLexiconTest.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import { assert, expect } from 'chai';
+import { run } from 'syncano-test';
+import helper from './util/testHelper';
+
+describe('Delete Lexicon', () => {
+  const config = helper.config;
+
+  it('returns information of requested celebrity', (done) => {
```
Comment (adamwardecki) [inline test/deleteLexiconTest.js]: Incorrect test name

File: test/putLexiconTest.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import { assert } from 'chai';
+import { run } from 'syncano-test';
+import helper from './util/testHelper';
```
Comment (adamwardecki) [inline test/putLexiconTest.js]: import { config, lexiconNames, lexiconFile } from './util/testHelper';

File: src/getLexicon.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
+import AWS from 'aws-sdk';
```
Comment (adamwardecki) [inline src/getLexicon.js]: This import is unnecessary.

File: src/util/helper.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+import Syncano from 'syncano-server';
+
+/** Class respresenting AWS rekognition actions */
+class helper {
+  /**
+   * Constructs a service object for each API operation
+   *
+   * @param {Object} configDetails - Contains the AWS configuration parameters
+   */
+  constructor(configDetails) {
```
Comment (adamwardecki) [inline src/util/helper.js:11]: you can destructure the configDetails in the constructor call

File: src/util/helper.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
+import AWS, { Polly } from 'aws-sdk';
+import Syncano from 'syncano-server';
+
+/** Class respresenting AWS rekognition actions */
+class helper {
```
Comment (adamwardecki) [inline src/util/helper.js]: Classes should be uppercased (Helper)

**Review by adamwardecki — CHANGES_REQUESTED** [review]
Looks good! Had couple of minor suggestions.

File: socket.yml [hunk 7]
```diff
@@ -0,0 +1,217 @@
...
+    required: true
+  AWS_SECRET_ACCESS_KEY:
+    description: AWS Access Secret Key
+    long_description: |
+      To find the key, log into your AWS account to get it
+    required: true
+  region:
+    description: Region
+    long_description: |
+      On your AWS Console, search for Rekognition to check supported regions and select one (e.g, us-east-1 )
```
Comment (adamwardecki) [inline socket.yml]: s/Rekognition/Amazon Polly

File: src/deleteLexicon.js (hunk 1 above)
Comment (chinazor-allen) [reply src/deleteLexicon.js]: done

File: src/getLexicon.js (hunk 4 above)
Comment (chinazor-allen) [reply src/getLexicon.js]: done

File: src/util/helper.js (hunk 6 above)
Comment (chinazor-allen) [reply src/util/helper.js]: done

File: test/deleteLexiconTest.js (hunk 2 above)
Comment (chinazor-allen) [reply test/deleteLexiconTest.js]: fixed

**Review by adamwardecki — APPROVED** [review]

**Comment by adamwardecki** [comment]
Please merge and publish the socket to the registry

**Pull request merged**